autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -60,9 +59,9 @@ def is_max_time_reached(self) -> bool:
 
 class RunSummary(object):
     def __init__(
-        self,
-        total_parameter_count: float,
-        trainable_parameter_count: float,
+            self,
```

Comment:
This is over indented if you're using pycharm you can fix it by going to setting > Editor > Cose style > python  and then change continuation indent to 4

Reply:
That is good to know, thanks